js/renovation/ui/scheduler/workspaces/base/date_table/all_day_panel/cell.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  Component, ComponentBindings, JSXComponent, OneWay,
+} from 'devextreme-generator/component_declaration/common';
+
+export const viewFunction = (viewModel: AllDayPanelCell) => (
+  <td
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    {...viewModel.restAttributes}
+    className={`dx-scheduler-all-day-table-cell dx-scheduler-cell-sizes-horizontal
+      dx-scheduler-first-group-cell dx-scheduler-last-group-cell
```

Comment:
These two classes should depend on a property (such as `isLastGroupCell`) because it may be used, for instance, with Week View